### PR TITLE
Feat/storybook gh pages

### PR DESCRIPTION
**Pull request by SNujiy**
New action added to github workflows to deploy storybook to [Gitlab page](https://ndla-h5p.github.io/h5p-editor-topic-map) when commits are made under `src/components`.

### CODE REVIEW

File: .github/workflows/storybook.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - name: Install and Build
+        run: |
+          npm install
+          npm run build-storybook
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@3.6.2
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: main
+          FOLDER: docs-build
```
Comment (boyum) [inline .github/workflows/storybook.yml:22]: Can this be set to `storybook-static`, as that's the original Storybook folder, and is already in `.gitignore`?

**Review by boyum — CHANGES_REQUESTED** [review]
We should remove the commited `docs` directory as part of this PR.

**Comment by boyum** [comment]
Looks good! ✨ Only a few comments.

**Review by boyum — APPROVED** [review]
Nice work!!

File: .github/workflows/storybook.yml (hunk 1 above)
Comment (SNujiy) [reply .github/workflows/storybook.yml]: Done :)

**Pull request merged**